.github/workflows/npm-publish.yml

```
@@ -0,0 +1,74 @@
+          git config user.name github-actions
+          git config user.email github-actions@github.com
+      
+      # Configure npm
+      - name: Configure npm
+        run: |
+          cat << EOF > "$HOME/.npmrc"
+            //registry.npmjs.org/:_authToken=$NPM_TOKEN
+          EOF
+        env:
+          NPM_TOKEN: ${{ secrets.APPSMITH_NPM_PUBLISH_KEY }}
+      
```

yamllint (1.35.1)

[error] trailing spaces

(trailing-spaces)